specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/InformationProtectionPolicies/CreateOrUpdateInformationProtectionPolicy_example.json

```diff
@@ -0,0 +1,238 @@
+{
+    "parameters": {
+        "api-version": "2017-08-01-preview",
+        "scope": "providers/Microsoft.Management/managementGroups/148059f7-faf3-49a6-ba35-85122112291e",
+        "informationProtectionPolicyName": "default",
+        "InformationProtectionPolicy": {
+            "id": "/providers/Microsoft.Management/managementGroups/148059f7-faf3-49a6-ba35-85122112291e/providers/Microsoft.Security/informationProtectionPolicies",
+            "name": "default",
+            "type": "Microsoft.Security/informationProtectionPolicies",
+            "properties": {
+                "sensitivityLabels": [
+                    {
+                        "id": "1345da73-bc5a-4a8f-b7dd-3820eb713da8",
```

Comment:
How would these GUIDs get generated in a ARM template?

Reply:
This is a string, user can set any id.
Our built-in labels have guids as id.
We'll update our examples to make it clear.

Reply:
Actually we decided that they should be guids, but these are determined by users and it can be any guid they choose.